dnstable.pyx

```diff
@@ -147,6 +147,65 @@ cdef class entry(object):
         free(res)
         return s
 
+    def to_json_rfc3339_time(self):
```

Comment:
It would be more flexible to have the "rfc3339_time" and "rdata_always_array" signalled by keyword arguments to the `to_json` method. In fact, I would advocate not exposing "formatter" through the python API at all, since the keyword argument approach should be sufficient, in addition to being more typical Python design.

Reply:
Done with to_json(self, rfc3339_time = False, rdata_always_array = False)